CMakeLists.txt

```diff
@@ -221,6 +246,15 @@ set(LIB_INCLUDE_DIRS
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
     $<INSTALL_INTERFACE:${CMAKE_INSTALL_FULL_INCLUDEDIR}>)
 
+# Define resource files for Apple Framework
+set(LIBDEFLATE_RESOURCES
+    "${CMAKE_CURRENT_SOURCE_DIR}/README.md"
+    "${CMAKE_CURRENT_SOURCE_DIR}/COPYING"
+    "${CMAKE_CURRENT_BINARY_DIR}/libdeflate-config.cmake"
+    "${CMAKE_CURRENT_BINARY_DIR}/libdeflate-targets.cmake"
+)
+message(STATUS "Resource files: ${LIBDEFLATE_RESOURCES}")
```

Comment:
Can you make this message conditional on LIBDEFLATE_APPLE_FRAMEWORK so that it doesn't show up for everyone else?